django/template/base.py

```diff
@@ -387,9 +387,7 @@ def create_token(self, token_string, position, lineno, in_tag):
                     self.verbatim = 'end%s' % block_content
                 return Token(TokenType.BLOCK, block_content, position, lineno)
             elif token_start == COMMENT_TAG_START:
-                content = ''
-                if token_string.find(TRANSLATOR_COMMENT_MARK):
```

Comment:
As `TRANSLATOR_COMMENT_MARK` is no longer used in `django/template/base.py`, we should move it into `django/utils/translation/template.py`.

Reply:
> Is there any backwards compatibility issue with doing this? I.e. is the availability of this import part of Django's public API?

Theoretically yes, we should add a reverse import for backward compatibility, or at least a small release notes.

Reply:
Actually, the `utils` file already depends on `base.py`, so it looks like it should be kept:
https://github.com/django/django/blob/910ecd1b8df7678f45c3d507dde6bcb1faafa243/django/utils/translation/template.py#L4

It's also where all the other string constants are kept, anyways.

Reply:
Yeah, I think it's worth keeping in the same place.

Reply:
I suggested the move as it isn't public API - it isn't documented. In addition it is translation-related, so moving it as suggested will keep translation-related code out of `django/template/base.py`. 🤷🏻‍♂️

Reply:
I was okay with moving it. I was just saying above that when I looked, I noticed it can't be moved with the current structure of the code since it would introduce a circular dependency.

Reply:
Yup. It's completely irrelevant to the module it's in with this change.